docs/docs/platform/graphql/index.md

```diff
@@ -1,39 +1,57 @@
 ---
 title: 'GraphQL'
-sidebar_position: 3
+sidebar_position: 1
 ---
 
-GraphQL is a query language for APIs. It provides a complete and understandable description of the data in your API.
+Every Nhost app has it's own autogenerated GraphQL API. The GraphQL API is based on the tables and columns in the [Postgres database](/platform/database) and is instanlty available. It's [Hasura GraphQL engine](https://github.com/hasura/graphql-engine) that powers the GraphQL API.
 
-Every Nhost app comes with a GraphQL API, and you can connect to this API with any GraphQL client you like. Most often you'd use the client included in [Nhost SDK](/reference/sdk/graphql).
+The GraphQL API is available at: `https://[subdomain].nhost.run/v1/graphql`.
+
+## What is GraphQL
+
+GraphQL is a query language for APIs that prioritize developer experience. The GraphQL API can be used to both fetch (query) and modify (mutation) data. GraphQL is particular powerful for frontend developers who wants to build products fast.
```

Comment:
"GraphQL is **particularly** (_question: **especially**?_) powerful for frontend developers..."